Blog post for watchlist roll-out (#375)
openSUSE/obs-landing: _posts/development/2022-09-26-watchlist-rollout.md

---
layout: post
title: The New Watchlist Came to Stay
category: development
---

We enabled the new watchlist feature in February 2022 under the beta program.
Since then, we have been improving this feature thanks to the valuable feedback coming from our users. Moving
forward today, we are now ready to push it out of the beta program and make it generally available in OBS.

> ### Advantages of the New Watchlist

Reviewer: ```suggestion
## Watch projects, packages and requests!
```

> We proposed the new watchlist feature in February 2022. We have been improving this
> feature since then. Thanks to our valuable users who started using this feature in
> beta and continuously helped us improve the feature with their feedback. Moving
> forward today, we are now ready to push it out of the beta program and make it available in OBS.

Reviewer 2: ```suggestion
We enabled the new watchlist feature in February 2022 under the beta program. Since then, we have been improving this feature thanks to the valuable feedback coming from our users. Moving
forward today, we are now ready to push it out of the beta program and make it generally available in OBS.
```

---
layout: post
title: The New Watchlist Came to Stay
category: development
---

We enabled the new watchlist feature in February 2022 under the beta program.
Since then, we have been improving this feature thanks to the valuable feedback coming from our users. Moving
forward today, we are now ready to push it out of the beta program and make it generally available in OBS.

### Watch projects, packages and requests!

OBS is huge and it is easy to lose track of important projects, packages, and requests.
The new watchlist solves this problem and helps you to keep track of your most used projects,
packages, and requests; unlike the old watchlist where you only could store projects.
The watchlist is available from every corner of the application,
which makes it easily accesible.

<figure>
  <img src="/images/posts/sprint_126_watchlist_rollout.png" alt="Watchlist" />
  <figcaption>Watchlist</figcaption>
</figure>

### Notifications and the new watchlist

> There are many options available in notification settings. The new watchlist ables you to receive

Reviewer 3: ```suggestion
There are many options available in notification settings. The new watchlist allows you to receive
```
